.NET Standard 2.0

stevehayles opened the pull request:
.NET Standard 2.0 is compatible with .NET Core 2.0, Full Framework 4.6.1, Unity 2018.1, Mono5.4, Xamarin.iOS 10.14 and Xamarin.Android 8.0

Full framework 4.5.2 support is effectively dropped in this commit

stevehayles committed:
Convert to new project format

Convert to a multi-targeted Netstandard and Net452 project for QuicNet, QuicNet.Utilities and QuicNet.Infrasturcture

stevehayles committed:
Convert Projects to NetStandard2.1

stevehayles committed:
Drop NetStandard version to 2.0

NetStandard 2.0 is compatible with .NET Core 2.0, Full Framework 4.6.1, Unity 2018.1, Mono5.4, Xamarin.iOS 10.14 and Xamarin.Android 8.0

.Net 4.5.2 support is effectively dropped in this commit

Vect0rZ commented:
@stevehayles It does give me error that the QuickNet.Test.ConsoleServer is targeting the .NET 4.5.2. Whenever you have a minute can you please up the version to 4.6.1 it seems to fix the issue. I can fix it as well, but It will override your pull request, and I want to keep your contribution clean :)

Thank you!

stevehayles committed:
Retarget samples to .NET 4.6.1

Update the ConsoleServer project to be compatible with the previous changes to target .NET Standard 2.0
